src/deepsparse/image_classification/schemas.py

```diff
@@ -33,6 +33,10 @@ class ImageClassificationInput(BaseModel):
     class Config:
         arbitrary_types_allowed = True
 
+    @classmethod
+    def from_files(cls, files):
```

Comment:
let's also change the type of `files` here to standardize to `List[str]`, that way we don't need to depend on `UploadFile` in non `server/` submodule code